derive_client/clients/base_client.py

```diff
@@ -140,155 +139,6 @@ def create_account(self, wallet):
             raise Exception(result_code["error"])
         return True
 
-    @safe
-    @validate_call
-    def deposit_to_derive_result(
```

Comment:
All bridging-related methods have been removed from `BaseClient`. They are now only exposed on `AsyncClient`, and via thin wrappers on `HttpClient`.

Additionally, the previous implementations were insufficient, as they did not allow users to inspect the prepared transaction before deciding whether to submit it.